tests/framework/utils_cpu_templates.py

```diff
@@ -23,16 +23,18 @@ def get_supported_cpu_templates():
     """
     Return the list of CPU templates supported by the platform.
     """
+    # pylint:disable=too-many-return-statements
     match cpuid_utils.get_cpu_vendor():
         case cpuid_utils.CpuVendor.INTEL:
             # T2CL template is only supported on Cascade Lake and newer CPUs.
-
             if global_props.cpu_codename == cpuid_utils.CpuModel.INTEL_SKYLAKE:
                 return sorted(set(INTEL_TEMPLATES) - set(["T2CL"]))
             return INTEL_TEMPLATES
         case cpuid_utils.CpuVendor.AMD:
             return AMD_TEMPLATES
         case cpuid_utils.CpuVendor.ARM:
+            if global_props.host_linux_version_tpl <= (6, 1):
+                return []
```

Comment:
It seems if `cpuid_utils.get_cpu_vendor() == cpuid_utils.CpuVendor.ARM` and `global_props.host_linux_version_tpl <= (6, 1)` then no CPU templates are supported? Since we only test kernels, 4.14, 5.10 and 6.1, wouldn't this disable CPU templates completely for ARM in our CI?

Reply:
Good catch it should be `<`, not `<=`

Reply:
I think it was initially `>=` and then I flipped the logic but messed up the operator.

Reply:
Done! Thanks for catching it!

Reply:
Validation that now it works as expected: https://buildkite.com/firecracker/firecracker-pr/builds/8665#018d3caa-7488-44ee-a123-ea76ca15f318/55-278